config/config.exs

```diff
@@ -64,6 +64,16 @@ config :archethic, :transaction_data_content_max_size, 3_145_728
 # 24KB Max
 config :archethic, :transaction_data_code_max_size, 24576
 
+config :archethic, :throttle,
+  by_ip: [
+    period: 1000,
+    limit: 150
+  ],
+  by_ip_and_path: [
+    period: 1000,
+    limit: 20
+  ]
```

Comment:
@bchamagne can you give me the numbers for the test you did to compare with the configuration ?

Reply:
or how to reproduce it locally

Reply:
this is normal, the website contains 266 files but the limit is at 150 no it's normal for it to crash

Reply:
I tried 
```
config :archethic, :throttle,
  by_ip: [
    period: 1000,
    limit: 300
  ],
  by_ip_and_path: [
    period: 1000,
    limit: 300
  ]
```
and I get the forbidden. People will not be able to upload to AEWeb if we don't relax the numbers